ui/src/utils/get_drag_event.rs

```diff
@@ -8,3 +8,5 @@ pub fn get_drag_event() -> FileDropEvent {
 
 pub static DRAG_EVENT: Lazy<RwLock<FileDropEvent>> =
     Lazy::new(|| RwLock::new(FileDropEvent::Cancelled));
+
+pub static BLOCK_CANCEL_DRAG_EVENT_FOR_LINUX: Lazy<RwLock<bool>> = Lazy::new(|| RwLock::new(false));
```

Comment:
if this only gets read from and written to in one place, this could also be an `AtomicBool`. Not that it matters much, just might be a little faster.